scripts/preinstall.js

```diff
@@ -1,19 +1,49 @@
+/* eslint-disable no-console */
 const fs = require('fs');
 const os = require('os');
 const path = require('path');
 
-const configsDir = path.resolve(os.homedir(), '.bpanel');
-const configsFile = path.resolve(configsDir, './config.js');
-const clientsDir = path.resolve(configsDir, 'clients');
+const BPANEL_DIR = path.resolve(os.homedir(), '.bpanel');
+const CONFIGS_FILE = path.resolve(BPANEL_DIR, './config.js');
+const CLIENTS_DIR = path.resolve(BPANEL_DIR, 'clients');
+const LOCAL_PLUGINS_DIR = path.resolve(BPANEL_DIR, 'local_plugins');
 
 const configText = `module.exports = {
   plugins: ['@bpanel/genesis-theme', '@bpanel/bui'],
   localPlugins: [],
 }`;
-if (!fs.existsSync(configsDir)) fs.mkdirSync(configsDir);
-if (!fs.existsSync(configsFile)) {
-  fs.appendFileSync(configsFile, configText);
-}
-if (!fs.existsSync(clientsDir)) fs.mkdirSync(clientsDir);
 
-require('./version.js');
+try {
+  if (!fs.existsSync(BPANEL_DIR)) {
+    console.log(
```

Comment:
Why `console.log` instead of `blgr`?

Reply:
since this is pre-install, we won't have any modules available at this point so everything in this file needs to be natively available in node

Reply:
👍  Maybe add a comment regarding that so we don't forget in the future?

Reply:
added